gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/PublisherSettings.java

```diff
@@ -223,7 +232,7 @@ protected PublisherSettings(MethodBuilders methods) {
   }
 
   /**
-   * Returns the ApiCallableBuilder for the API method createTopic.
+   * Returns the builder for the API method createTopic.
```

Comment:
Did I mention that I think you should consider renaming `ApiCallableBuilder` to `Builder`? As a nested class I think
its context is clear and such naming convention is very common ([e.g.](https://github.com/google/guava/blob/master/guava/src/com/google/common/collect/ImmutableList.java#L612)).